### ISSUE TITLE
When using both ScrollX and ScrollY in the Table component, there may be a situation where the header cells and content cells cannot be aligned when the properties are used

### ISSUE THREAD

**Issue by wangfei9999** [ant-design-blazor/ant-design-blazor#4203]
Bug Example 1：
```
<Table DataSource="@data"
       Bordered
       HidePagination
       ScrollX="1000"
       ScrollY="200">
    <PropertyColumn Property="c=>c.Name" Width="100"></PropertyColumn>
    <PropertyColumn Property="c=>c.Money" Width="100"></PropertyColumn>

    <PropertyColumn Property="c=>c.Address" Width="100"></PropertyColumn>
    <PropertyColumn Property="c=>c.Address" Width="100"></PropertyColumn>

    <PropertyColumn Property="c=>c.Address" Width="100"></PropertyColumn>
    <PropertyColumn Property="c=>c.Address" Width="100"></PropertyColumn>

    <PropertyColumn Property="c=>c.Address" Width="100"></PropertyColumn>
    <PropertyColumn Property="c=>c.Address" Width="100"></PropertyColumn>

    <PropertyColumn Property="c=>c.Address" Width="100"></PropertyColumn>
    <PropertyColumn Property="c=>c.Address" Width="100"></PropertyColumn>
</Table>


@code {
    Data[] data = Enumerable.Range(1, 100).Select(n => new Data()
        {
            Key = n.ToString(),
            Name = "John Brown",
            Money = "￥300,000.00",
            Address = "New York No. 1 Lake Park",
        }).ToArray();

    public class Data
    {

        [DisplayName("Key")]
        public string Key { get; set; }


        [DisplayName("Name")]
        public string Name { get; set; }


        [DisplayName("Money")]
        public string Money { get; set; }


        [DisplayName("Address")]
        public string Address { get; set; }
    }
}
```

**Comment by wangfei9999** [comment]
将ScrollX更改为2000，并且每个列宽设置为200后，又神奇的出现对齐了！！！
```
<Table DataSource="@data"
       Bordered
       HidePagination
       ScrollX="2000"
       ScrollY="200">
    <PropertyColumn Property="c=>c.Name" Width="200"></PropertyColumn>
    <PropertyColumn Property="c=>c.Money" Width="200"></PropertyColumn>

    <PropertyColumn Property="c=>c.Address" Width="200"></PropertyColumn>
    <PropertyColumn Property="c=>c.Address" Width="200"></PropertyColumn>

    <PropertyColumn Property="c=>c.Address" Width="200"></PropertyColumn>
    <PropertyColumn Property="c=>c.Address" Width="200"></PropertyColumn>

    <PropertyColumn Property="c=>c.Address" Width="200"></PropertyColumn>
    <PropertyColumn Property="c=>c.Address" Width="200"></PropertyColumn>

    <PropertyColumn Property="c=>c.Address" Width="200"></PropertyColumn>
    <PropertyColumn Property="c=>c.Address" Width="200"></PropertyColumn>
</Table>
```

**Comment by ElderJames** [comment]
Thanks for contacting us @wangfei9999 , this issue have been fixed via #4195
